pkg/api/v1/conversion.go

```diff
@@ -24,12 +24,22 @@ import (
 	"k8s.io/kubernetes/pkg/conversion"
 )
 
+const (
+	// Annotation key used to identify mirror pods.
+	mirrorAnnotationKey = "kubernetes.io/config.mirror"
+
+	// Value used to identify mirror pods from pre-v1.1 kubelet.
+	mirrorAnnotationValue = "mirror"
```

Comment:
Name isn't suggestive enough about the purpose. I suggest something like mirrorAnnotationValue_1_0